aws-lambda-java-events/pom.xml

```diff
@@ -80,6 +80,32 @@
         </dependency>
     </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.Final</version>
```

Comment:
Latest version seems to be 1.2.2, could you please update it?
https://mvnrepository.com/artifact/org.moditect/moditect-maven-plugin/1.2.2.Final